Tools/CISupport/ews-app/ews/common/github.py

```diff
@@ -236,7 +237,7 @@ def generate_comment_text_for_change(self, change):
         regular_comment = u'{}{}'.format(hash_url, comment)
         folded_comment = u'EWS run on current version of this PR (hash {})<details>{}</details>'.format(hash_url, comment)
         if change.comment_id == -1:
-            pr_url = GitHub.pr_url(change.pr_id)
+            pr_url = GitHub.pr_url(change.pr_id, repository_url=repository_url)
             folded_comment = u'Starting EWS tests for {}. Live statuses available at the PR page, {}'.format(hash_url, pr_url)
```

Comment:
Is this change only for the url in this email notification?

If so, we should consider just removing this logic of this custom message of `Starting EWS tests for ...`. It anyways doesn't work most of the time because of race conditions.

Reply:
That was the situation that prompted this fix, yes.

I think we move forward and land this, then discuss removing the logic. It's needed to be pedantically correct, and I can see a future where, for example, this code gets mutated to include a PR link in every email.

Reply:
ok, sounds good